<launch>
    <include file="$(find bitbotsQuinticWalk)/launch/quintic_walk_standalone.launch"/>

    <node name="hcm_bypass" pkg="bitbotsQuinticWalk" type="hcm_bypass.py" respawn="false" output="screen" args=""/>

</launch>
